client/src/components/Loading/Loading.js

```diff
@@ -3,7 +3,7 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 
 /**
- * Renders a SilverStripe loading animation
+ * Renders a Silverstripe loading animation styled after the Silverstripe logo
```

Comment:
Just being a little more explicit here - I had to render it out to find out what it was, especially since it uses the terminology "spinner" in the css classnames.